README.md

```diff
@@ -30,7 +30,7 @@ Bloxstrap is only supported for PCs running Windows.
 
 **Q: Is this malware?**
 
-**A:** No. The source code here is viewable to all, and it'd be impossible for us to slip anything malicious into the downloads without anyone noticing. Just be sure you're downloading it from an official source. The only two official sources are this GitHub repository and [bloxstraplabs.com](https://bloxstraplabs.com).
+**A:** No. The source code here is viewable to all, and it'd be impossible for us to slip anything malicious into the downloads without anyone noticing. Just be sure you're downloading it from an official source. The only 3 official sources are this GitHub repository, [bloxstraplabs.com](https://bloxstraplabs.com) and [Winget](https://winstall.app/apps/pizzaboxer.Bloxstrap).
```

Comment:
You should link to the release page for the "GitHub repository" in case they are looking for the source code download, relevant to the question.

Very minor nitpicking but [3 should be spelled out](https://www.grammarly.com/blog/writing-tips/when-to-spell-out-numbers/).

```suggestion
**A:** No. The source code here is viewable to all, and it'd be impossible for us to slip anything malicious into the downloads without anyone noticing. Just be sure you're downloading it from an official source. The only three official sources are this [GitHub repository](https://github.com/bloxstraplabs/bloxstrap), [bloxstraplabs.com](https://bloxstraplabs.com) and [Winget](https://winstall.app/apps/pizzaboxer.Bloxstrap).
```

Reply:
i don't think theres a point in linking anything at all since you're already at the github repo, and you don't need to visit the releases page to download the source code.

Reply:
> i don't think theres a point in linking anything at all since you're already at the github repo, and you don't need to visit the releases page to download the source code.

Fair enough. I'll keep my edits in the review just in case if they want to implement it though.